stdeb.cfg

```diff
@@ -6,7 +6,7 @@ Depends:
     ipython (>= 2.0),
     python-traits,
     python-traitsui,
-    python-nose,
+    python-pytest,
     ipython-qtconsole,
     ipython-notebook,
     python-h5py,
```

Comment:
Just out of curiosity, why some package are python-something and others are python3-something?